cmake/FindCUDA/run_nvcc.cmake

```diff
@@ -149,7 +149,7 @@ macro(cuda_execute_process status command)
     execute_process(COMMAND ${CMAKE_COMMAND} -E echo ${cuda_execute_process_string})
   endif()
   # Run the command
-  execute_process(COMMAND ${ARGN} RESULT_VARIABLE CUDA_result )
+  execute_process(COMMAND ${OPENCV_NVCC_LAUNCHER} ${ARGN} RESULT_VARIABLE CUDA_result )
```

Comment:
This file is a standalone build script.
Please check that this value is passed here properly (command line or `@CUDA_HOST_FLAGS@` above)

Reply:
I'm still struggling here.